assets/stylesheets/style.scss

```diff
@@ -0,0 +1,946 @@
+.bubble.bot-avatar::after,
+.bubble.-response::after {
+  content: '';
+  position: absolute;
+  height: 10px;
+  width: 20px;
+  background-color: #e2ebe9;
+  bottom: 7px;
+}
+
+.bubble.bot-avatar::after {
+  left: -10px;
+}
+
+.bubble.-response::after {
+  background-color: #1793ad;
+  right: -8px;
+}
+
+//WHITE CIRCLE TO COVER GRAY ARROW
+.bot-avatar > .spinner::after,
+.bot-avatar > p::after,
+.bubble.-response > p::after {
+    content: '';
+    position: absolute;
+    bottom: 8px;
+    height: 20px;
+    width: 20px;
+    background-color: white;
+    border-radius: 100px;
+    z-index: 1;
+
+}
+.bot-avatar > .spinner::after, .bot-avatar > p::after, {
+    left: -20px;
+}
+
+.bubble.-response {
+  p::after {
+    right: -20px;
+    bottom: 7px;
+  }
+}
+
+//CHAT BUBBLES && SPINNER
+@keyframes appear-text {
+  0% {
+    top: -10px;
+    transform: translateY(20px) translateX(-30px) scale(0.5)
+  }
+  50% {
+    top: 3px;
+  }
+  100% {
+    opacity: 1;
+    top: 0px;
+    transform: translateY(0px) translateX(0px) scale(1)
+  }
+}
+
+@keyframes appear-text-right {
+  0% {
+    top: -10px;
+    transform: translateY(20px) translateX(30px) scale(0.5)
+  }
+  50% {
+    top: 3px;
+  }
+  100% {
+    opacity: 1;
+    top: 0px;
+    transform: translateY(0px) translateX(0px) scale(1)
+  }
+}
+
+
+.bubble.flo > p {
+  opacity: 0;
+}
+
+.bubble.flo.finished {
+  .spinner {
+    visibility: hidden;
+  }
+  p {
+    opacity: 1;
+  }
+}
+
+
+
+.bubble {
+  position: relative;
+  margin-bottom: 7px;
+  animation: appear-text 0.2s ease-out forwards paused;
+  animation-iteration-count: 1;
+  transform-origin: left bottom;
+  opacity: 0;
+
+
+  p, .spinner {
+    line-height: 1.375;
+    color: #252224;
+    display: inline-block;
+    border-radius: 15px;
+    background-color: #e2ebe9;
+    padding: 12px;
+    margin: 0;
+  }
+
+  .spinner {
+    position: absolute;
+    bottom: 0
+  }
+
+  .spinner > span {
+    width: 8px;
+    height: 8px;
+    background: #514D4C;
+    display: inline-block;
+    border-radius: 100px;
+    animation: typing-dots 1s ease-in infinite;
+  }
+
+  .spinner > span:nth-child(2) {
+    animation-delay: 0.2s;
+  }
+  .spinner > span:nth-child(3) {
+    animation-delay: 0.4s
+  }
+}
+
+.bubble.-response {
+  animation-name: appear-text-right;
+  transform-origin: right bottom;
+}
+
+@keyframes typing-dots {
+  0% {
+    background: #cacaca
+  }
+  60% {
+    background: #514D4C
+  }
+}
+
+
+.bubble.-response {
+  align-self: flex-end;
+
+  p {
+    background-color: #1793ad;
+    color: white
+  }
+}
+
+.viget {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  div {
+    background: url("../../public/img/viget.png");
+    width: 125px;
+    height: 62px;
+    position: relative;
+    right: -30px;
+  }
+}
+
+.viget > div::before {
+  content: url('../../public/img/viget_2.png');
+  position: absolute;
+  left: -70px;
+}
+
+
+
+@media screen and (min-width: 440px) {
```

Comment:
We prefer to nest media queries so you can see all the modifications on an element in one place, so:

```scss
header {
   margin: 0 auto;
   max-width: 700px;
   
   @media screen and (min-width: 440px) {
      margin-bottom: 50px;
   }
}
```

Scss will handle moving the media query outside of the selector. And there's a great [build tool plugin](https://www.npmjs.com/package/css-mqpacker) that can organize and merge media queries.